docs/VERSION_UPDATE_CHECKLIST.md

```
@@ -64,46 +64,148 @@ calibre-debug -g
-# 遵循之前 CHANGELOG 的格式，之后要手动推送到 mobileread.com 和 GitHub 的 release 页面
+# 2. 更新 VERSION_UPDATE_GUIDE.md
+# 在"版本号历史记录"中添加新版本记录
 ```
 
 ---
 
 ## ✅ 完成检查
 
+### 代码更新
 - [ ] version.py 已更新
 - [ ] __init__.py Line 22 已更新（🔴 最重要）
```

markdownlint-cli2 (0.18.1)

Strong style
Expected: asterisk; Actual: underscore

(MD050, strong-style)